### ISSUE TITLE
Review source string comments in Weblate (German)

### ISSUE THREAD

**Issue by dorsiflexion** [andersju/webbkoll#32]
Hello,

there are a few (about 5–10) source string comments in Weblate, i. e. I had questions regarding some original/English strings. Some are very recent while others are over a year old and still waiting for a response. So I thought I'd give you a quick notification here.

I would greatly appreciate it if you answered those comments/questions. It would help improve the quality of the translation and maybe even of the original strings.

**Comment by andersju** [comment]
Hi,

Thanks for the notification. I responded to the ones in English. (I don't speak German so can't help with those.)

**Comment by dorsiflexion** [comment]
Thanks a lot!

The ones in German should be translation comments and are meant for potential other translators.

**Comment by dorsiflexion** [comment]
Sorry, I just noticed my newest comments are not shown if you search for comments in the German translation. They can be seen if you [search for comments on English strings](https://weblate.dataskydd.net/translate/webbkoll/main/en/?q=has:comment). These are only 3 more (one has already been responded to).

**Comment by andersju** [comment]
Apologies for my slowness here. I think I just took care of the last string you had a comment on now (06f1f28). Please let me know if there's anything I missed.

**Comment by dorsiflexion** [comment]
No problem. Now I was slow on my side. Since the comments are gone (and my account, too), I cannot easily find these strings again. Let's just consider this as solved.

**Comment by andersju** [comment]
Sorry about the lack of a heads up. Moved from self-hosted to "Hosted Weblate" recently (same service as a lot of other free software projects), so no account information was transferred (though of course the git log of this repo contains all translation contributions).

I did look at the old installation just now but couldn't find much in the way of unresolved (English) comments, because most were gone anyway - Weblate ([by design](https://github.com/WeblateOrg/weblate/issues/7189)) removes comments if the associated string is changed/removed. I know I did 13000d3cb11864dc1ed22052107a70f081e2b13e and 06f1f2810ec599f2280e179a90ea7c347a7ca2d9 recently-ish based on your comments.
